spec/computer_spec.rb

```diff
@@ -1,73 +1,75 @@
 require 'spec_helper'
-require 'computer'
+
 require 'board'
+require 'player'
+require 'computer'
+require 'marks'
 
 RSpec.describe Computer do
   let (:input) { StringIO.new("1\n") }
   let (:output) { StringIO.new }
-  let (:ui) { CliInterface.new(input, output, "spec/lang/") }
```

Comment:
Glad to see the UI dependency removed from the `Computer` player! 👍 
Not sure why that is part of a PR that starts with extracting a binary, though. ;)